modin/pandas/test/utils.py

```diff
@@ -331,7 +331,7 @@
     "sum of certain elements": lambda axis: (
         axis.iloc[0] + axis.iloc[-1] if isinstance(axis, pandas.Series) else axis + axis
     ),
-    "should raise TypeError": 1,
+    "should raise AssertionError": 1,
```

Comment:
This is more similar to the errors that I have seen. Can be called more neutrally (`should raise Exception`).